sdk/cosmos/azure-cosmos/README.md

```diff
@@ -588,6 +584,46 @@ def integrated_cache_snippet():
 ```
 For more information on Integrated Cache, see [Azure Cosmos DB integrated cache - Overview][cosmos_integrated_cache].
 
+### Using Transactional Batch
+Transactional batch requests allow you to send several operations to be executed at once within the same partition key.
+If all operations succeed in the order they're described within the transactional batch operation, the transaction will be committed.
+However, if any operation fails, the entire transaction is rolled back.
+
+Transactional batches have a limit of 100 operations per batch, and a total size limit of 1.2Mb for the
+batch operations being passed in.
+
+Transactional Batch operations look very similar to the singular operations apis, and are tuples containing
+(`operation_type_string`, `args_tuple`, `optional_kwargs_dictionary`):
+```python
+batch_operations = [
+        ("create", (item_body,), kwargs),
+        ("replace", (item_id, item_body), kwargs),
+        ("read", (item_id,), kwargs),
+        ("upsert", (item_body,), kwargs),
+        ("patch", (item_id, operations), kwargs),
+        ("delete", (item_id,), kwargs),
+    ]
+batch_results = container.execute_item_batch(batch_operations=batch_operations, partition_key=partition_key)
+```
+In this case, the kwargs for each operation can include `filter_predicate` in the case of using conditional patching, or
+the use of the `if_match_etag`/`if_none_match_etag` if you'd like to pass an etag filter for one single operation:
+```python
+batch_operations = [
+        ("replace", (item_id, item_body), {"if_match_etag": etag}),
```

Comment:
If looks like this doesn't actually match the kwargs that I would pass into the replace method?
Are we instead using this dict as basically raw request headers?

Reply:
in this case I simplified the e_tag use, but you're right we should remain consistent